src/components/AltinnAppHeader.module.css

```diff
@@ -0,0 +1,95 @@
+.appBarWrapper {
+  flex-grow: 1;
+}
+
+.appBarWrapper header {
+  box-shadow: none;
+}
+
+.blueDark {
+  color: #0062ba;
+}
+
+.blueDarker {
+  color: #022f51;
+}
```

Comment:
I see there are som css tokens from the `"altinn/figma-design-tokens"` and in `"digdir/design-system-tokens"` packages that we could use. 

I am guessing the theme is on its way out as we are removing more and more of the MUI `makeStyles()` styling, since the theme's variables are defined in typescript and not accessible directly in the css files. 

Should we depend on the figma-tokens from either the Designsystem or the old Altinn-designsystem, or should we find some new way of handling our 'theme' internally?

Reply:
Yup, I'd rather we depend on those from the new design system. 😊 I guess we could also create our own file to re-export some variables so that our various CSS modules don't all depend on the design system tokens directly, but we can do that in the future at some point if the current situation gets problematic.